tfx_bsl/beam/run_inference.py

```diff
@@ -383,14 +504,20 @@ def setup(self):
     # user agent once custom header is supported in googleapiclient.
     self._api_client = discovery.build('ml', 'v1')
 
+  def _extract_from_recordBatch(self, elements: pa.RecordBatch):
+    serialized_examples = bsl_util.ExtractSerializedExampleFromRecordBatch(elements)
```

Comment:
Seems this is the same in Batch and Remote DoFn. Maybe extract this out to Base, and only get model_input in _extract_from_recordBatch?